app.vue

```diff
@@ -0,0 +1,27 @@
+<script setup>
+const { data } = await useAsyncData('data', () => queryContent().findOne());
+const sortedByDate = data.value.body.sort((a, b) => (a.date > b.date) ? 1 : (a.date < b.date) ? -1 : 0)
+
+useSeoMeta({
+  title: 'Общее расписание на ' + data.value.title.toLowerCase(),
+  description: 'Общее расписание обучающих семинаров и вебинаров компании Фармацевтикал Косметик',
+});
+</script>
+
+<template>
+  <Body>
+    <Header />
+    <main class='flex flex-col gap-6 desktop:gap-8 px-4 tablet:px-6 mb-12 desktop:mb-20'>
```

Comment:
[Minor]
Take the comment with a grain of salt, as I'm a backend developer.
I think that giving names to most tags/DOM nodes and creating many small components can help to make the structure easier to understand.

For example:
```html
<template>
  <WebinarLandingPage>
    <Header />
    <PageContent />
    <Footer />
  </WebinarLandingPage>
</template>
```